### ISSUE TITLE
Creation of a new user unaccessible on the training VM

### ISSUE THREAD

**Issue by sparetamol** [TheHive-Project/TheHiveDocs#54]
Hi,

When trying to use the training VM (both the latest one and the previous one), after the installation and with a local authentication scheme, after clicking the "update database" button on the maintenance page, it skips the creation of the new user and goes directly to the login form. It is hence not possible to login into the application because no user exist.

Could this be an issue or an error I made while installing or configuring TheHive? Thanks for your help !

**Comment by jeromeleonard** [comment]
Hi, 

I can't reproduce the problem. Running it on VMware Fusion on MacOS. 
Could you please give more details on your environment ?

**Comment by sparetamol** [comment]
Hi,

Thank's for your reply.

I am on CentOS 7.4. I used both the latest training VM and the previous one from this link: https://github.com/TheHive-Project/TheHiveDocs/blob/master/training-material.md
I used a local authentication scheme. TheHive and the Elasticsearch database are on the same host.

Until this morning, I was using Firefox only (v52.2 64 bits). I tried with Google Chrome (v67.0 64 bits) and it worked (updating the database and creating a new user). So, there might be a compatibility issue with Firefox, even so I was not able to find which one.
